tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.robot

```diff
@@ -0,0 +1,166 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 5-22 - NFS Volume
+Resource  ../../resources/Util.robot
+Suite Setup  Setup NFS Server
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+
+*** Variables ***
+${createFileContainer}=  createFileContainer
+${addToFileContainer}=  addToFileContainer
+
+
+*** Keywords ***
+Setup NFS Server
+    Log To Console  \nStarting test...
+    ${esx3}  ${esx4}  ${esx5}  ${vc}  ${esx3-ip}  ${esx4-ip}  ${esx5-ip}  ${vc-ip}=  Create a Simple VC Cluster  datacenter1  cls1
```

Comment:
Using just an ESX would speed things up quite a bit.  I don't think the use case with a VC adds any value to this test.

I'll redo this using just an ESX setup.